src/main/resources/l10n/Menu_es.properties

```diff
@@ -56,7 +56,7 @@ Paste=&Pegar
 Preferences=&Preferencias
 Previous_tab=&Pestaña_anterior
 Quit=&Salir
-Recent_files=Bases_de_datos_&recientes
+Recent_databases=
```

Comment:
I think if we leave the old translations like they are now, no one feels like imediatly updating the translation. By removing them completly the need to change ist bigger.